kafka-rest/src/main/java/io/confluent/kafkarest/resources/v2/ConsumersResource.java

```diff
@@ -182,7 +183,7 @@ public void readRecordBinary(
         asyncResponse,
         group,
         instance,
-        timeout,
+        Duration.ofMillis(timeout),
```

Comment:
`timeout_ms`, not `timeout` is a better query param name imo.

Reply:
Changed the names of the variables. Unfortunately we can't change the name of the query params, since that would not be backwards compatible.